server/command.go

```diff
@@ -125,7 +126,7 @@ func (p *Plugin) getCommandResponse(args *model.CommandArgs, text string) *model
 }
 
 // ExecuteCommand is the entrypoint for /gitlab commands. It returns a message to display to the user or an error.
-func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
+func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (res *model.CommandResponse, err *model.AppError) {
```

Comment:
One suggestion to name the `model.AppError` as `appErr`, to differentiate from regular `error`s below. It looks like those are all reclared via `:=`, though there can potentially be pitfalls of having a concrete type error, and interface error in the same scope. Not an issue right now (and maybe the compiler would catch any issues with code introduced later), though we typically name it `appErr` also for readability.

```suggestion
func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (res *model.CommandResponse, appErr *model.AppError) {
```